src/frontend/webapp.js

```diff
@@ -281,9 +281,9 @@ app.post(config.api.requestUserConfirmUrl, function (req, res) {
             //var auth_details = require('./static/email2/Authentication.json');
             //if (err) {throw err}
             //var transporter = nodemailer.createTransport(auth_details.user + ':' + auth_details.pass);
-            //var transporter = nodemailer.createTransport('smtps://xiang.liu1995@gmail.com:19951220lx@@smtp.gmail.com');
-            var transporter = nodemailer.createTransport('smtps://genamap.v2.0@gmail.com:GenAMapV2@smtp.gmail.com')
-            //var transporter = nodemailer.createTransport('smtps://email@gmail.com:pass@smtp.gmail.com');
+            //var transporter = nodemailer.createTransport('smtps://genamap.v2.0@gmail.com:GenAMapV2@smtp.gmail.com')
+            //var transporter = nodemailer.createTransport('smtps://wentingye52@gmail.com:1996111ywt@smtp.gmail.com')
+            var transporter = nodemailer.createTransport('smtps://email@gmail.com:pass@smtp.gmail.com');
```

Comment:
how does this get the authentication details? We don't want to write them in the repo in plaintext, which is why we previously used Authentication.json (not checked in to the repo).